src/guards/isUploadedFile.ts

```diff
@@ -2,11 +2,11 @@ import { isPlainObject } from 'is-plain-object';
 
 import type { UploadedFile } from '../types';
 
-export function isUploadedFile(file: any | undefined): file is UploadedFile {
+export function isUploadedFile(file: Record<string, any> | undefined | null): file is UploadedFile {
     return (
         typeof file === 'object' &&
         isPlainObject(file) &&
-        typeof file.uuid === 'string' &&
+        typeof file?.uuid === 'string' &&
```

Comment:
Hm... why is this necessary? Isn't `isPlainObject` already checking for `file` object to be not null?

Reply:
It won't complain if `file` will stay typed as `any`.

Technically speaking, `file: Record<string, any> | undefined | null` type is not correct. As the type-guard checks any value. Not just `null | undefined | object`.

Reply:
`any` is a super type, adding "any | null | undefined" is the same as just "any". Because of that we have got an issue, `typeof null` is object, and if TS would know that the argument could be null it will complain about that and forbid accessing properties